downstream/modules/platform/ref-controller-system-requirements.adoc

```diff
@@ -0,0 +1,94 @@
+[id="ref-controller-system-requirements"]
+
+== {ControllerNameStart} system requirements
+
+{ControllerNameStart} is a distributed system, where different software components can be co-located or deployed across multiple compute nodes.
+In the installer, node types of control, hybrid, execution, and hop are provided as abstractions to help you design the topology appropriate for your use case.
+
+The following tables provides recommendations for node sizing:
```

Comment:
provides > provide*

To avoid self references, I would suggest "Apply/Use the following recommendations for node sizing"